### DISCUSSION TITLE
Handover support

### DISCUSSION THREAD

**Discussion by dadaWilliam** [srsran/srsRAN_Project#547] [General Help]
Hi, I have found several related discussions and in one discussion, it is said that the handover will be released soon, May I kindly ask when it will be released and any tutorials on how to achieve such function?

THX!!!

<img width="914" alt="image" src="https://github.com/srsran/srsRAN_Project/assets/90446055/08c18fc6-dd86-49c1-a73c-67aa5225ebda">

**Comment by andrepuschmann** [comment]
Situation is still the same. We are preparing it for the 24.4 release and will also deliver a application notes along with it. Stay tuned for this we make sure to announce it properly.

**Reply by yc541** [reply]
I just noticed 24.4 was pushed to main yesterday with the F1 handover supported. Is there a link to the application notes or is it not available yet? Look forward to testing the handover capability. Many thanks!
